aten/src/ATen/native/native_functions.yaml

```diff
@@ -363,6 +363,9 @@
 - func: type_as(Tensor self, Tensor other) -> Tensor
   variants: method
 
+- func: unique(IndexTensor self, bool sorted=false, bool return_inverse=false) -> (Tensor, Tensor)
+  variants: method
```

Comment:
why is it only a method?

Reply:
No reason - just wasn't sure what the convention for method vs function (or both) was. Added function